File: 242-valid-anagram/valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int> arr(26, 0);
        for(auto ch : s) {
            arr[ch-'a']++;
        }
        for(auto ch : t) {
            arr[ch-'a']--;
        }
        for(auto a : arr) {
            if(a) return false;
        }
        return true;
    }
};

File: 242-valid-anagram/valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        int[] map = new int[26];
        for(char ch : s.toCharArray()) {
            map[ch-'a']++;
        }
        for(char ch : t.toCharArray()) {
            map[ch-'a']--;
        }
        for(int i : map) if(i != 0) return false;
        return true;
    }
}